apps/web/src/features/organizations/components/Dashboard/index.tsx

```diff
@@ -29,41 +34,84 @@ const SignedOutState = () => {
   )
 }
 
-const OrganizationsDashboard = ({ organizationId }: { organizationId: string }) => {
-  // TODO: use the organizationId to fetch the organization data
-  console.log('organizationId', organizationId)
+const ViewAllLink = ({ url }: { url: string }) => {
+  return (
+    <NextLink href={url} passHref legacyBehavior>
+      <Link
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          gap: 1,
+          textDecoration: 'none',
+          fontSize: '14px',
+          color: 'primary.main',
+        }}
+      >
+        View all <ChevronRightIcon fontSize="small" />
+      </Link>
+    </NextLink>
+  )
+}
 
-  if (!isSignedIn) {
+const OrganizationsDashboard = () => {
+  const safes = useOrgSafes()
+  const isUserSignedIn = useAppSelector(isAuthenticated)
+  const orgId = useCurrentOrgId()
+
+  if (!isUserSignedIn) {
     return <SignedOutState />
   }
 
   return (
     <>
-      <Typography variant="h1" fontWeight={700} mb={4}>
-        Getting started
-      </Typography>
+      {safes.length > 0 ? (
+        <>
+          <Typography variant="h1" fontWeight={700} mb={4}>
+            Dashboard
+          </Typography>
+
+          <Grid container spacing={3}>
+            <Grid size={{ xs: 12, md: 8 }}>
+              <Stack direction="row" justifyContent="space-between" alignItems="center" mb={3}>
+                <Typography variant="h5">Safe Accounts ({safes.length})</Typography>
+                {orgId && <ViewAllLink url={AppRoutes.organizations.safeAccounts(orgId)} />}
+              </Stack>
+
+              <SafesList safes={safes} isOrgSafe />
```

Comment:
It would probably be good if we limited the number of safes that we show on the dashboard i.e. via a prop. I suggest to add a TODO here to not forget about it.